integration-tests/tests/analysis-context-bar.js

```diff
@@ -63,5 +64,5 @@ registerTest({
   name: 'analysis-context-bar',
   fn: testAnalysisContextBarFn,
   timeout: 15 * 60 * 1000,
-  targetEnvironments: []
+  targetEnvironments: ['dev']
```

Comment:
`targetEnvironments: ['dev']` means this test will never be run. Integration tests against dev run with environment set to local.

https://github.com/DataBiosphere/terra-ui/blob/6dae7a5cbf8bb482e52aa1aceaf290c9e11366bf/.circleci/config.yml#L268-L274